### PR TITLE
Add 'aframe-react-stories' to Scenes

### PR DESCRIPTION
**Pull request by aschnapp**

### CODE REVIEW

**Comment by donmccurdy** [comment]
Hi @aschnapp! The second link (http://vrstories.aschnapp.com) seems to be broken.

File: README.md [hunk 1]
```diff
@@ -224,7 +224,6 @@ Integration with existing tools, frameworks, and libraries.
 - [aframe-next-static](https://github.com/michaltakac/aframe-next-static) - This boilerplate leverages extremely simple deployment flow from Zeit's Now, now available for WebVR with A-Frame and React
 
 ### Scenes
-
```
Comment (cvan) [inline README.md]: can you please keep this?

File: README.md [hunk 2]
```diff
@@ -319,7 +318,7 @@ Integration with existing tools, frameworks, and libraries.
 - [Temples of Cambodia](http://interactive.aljazeera.com/aje/2017/cambodia-temples/) - Tour of the most popular temples of Cambodia, from [Al Jazeera](http://www.aljazeera.com/)
 - [React Imago3D](https://github.com/kenta88/react-imago3d) - Make WebVR easily creating your own scene with built-in 3d object and use your browser or vr headset to see what you built in real time.
 - [Have fun with nature](https://polideaplayground.github.io/WebVRExperience/) - Play with different environmental effects using your desktop browser or mobile VR headsets, like Daydream, Gear VR or Cardboard. [[Code]](https://github.com/PolideaPlayground/WebVRExperience)
-
+- [aframe-react-stories demo](http://afsdemo.aschnapp.com) - Demo of [aframe-react-stories](http://npmjs.com/aframe-react-stories) module. A module to quickly add social media style stories in VR to your project.
```
Comment (cvan) [inline README.md]: can you use `https:` (not `http:`) URLs?

Comment (cvan) [inline README.md]: I would move this from `Scenes` to `Components` section, and perhaps name it as `React Stories` (or just `Stories`) component

**Review by cvan — CHANGES_REQUESTED** [review]
thanks for submitting - just few changes :)

File: README.md [hunk 3]
```diff
@@ -178,6 +178,7 @@ components, see the [A-Frame Registry](https://aframe.io/aframe-registry/).
 - [Star](https://github.com/andreasplesch/aframe-star-component) - 2D stars
 - [Stereo](https://github.com/oscarmarinmiro/aframe-stereo-component) - Enable rendering different entities for each eye, using three.js layers so you can render 3D stereoscopic videos (full and half-dome)
 - [Stereo Cube](https://github.com/wallabyway/aframe-stereocube) - Use stereoscopic cube-map textures (i.e., left and right eye textures)
+- [Stories](https://afsdemo.aschnapp.com) - Demo of [aframe-react-stories](https://npmjs.com/aframe-react-stories) module. A module to quickly add social media style stories in VR to your project.
```
Comment (cvan) [inline README.md]: I would change the URL to `https://www.npmjs.com/package/aframe-react-stories` (since that's what it redirects to

Comment (cvan) [reply README.md]: thanks for moving and tweaking this.

what do you think of this text?

```
- [Stories](https://afsdemo.aschnapp.com) - Quickly add social media-style stories in VR to your React-based project. [[Demo]](https://www.npmjs.com/package/aframe-react-stories)
```

**Review by cvan — CHANGES_REQUESTED** [review]
thanks for updating! looking good - let me know your thoughts on the summary rewording.

**Comment by ngokevin** [comment]
We've discontinued awesome. Thanks!

**Pull request closed without merging**